Library/Formula/open-mesh.rb

```diff
@@ -16,11 +16,17 @@ class OpenMesh < Formula
   head "http://openmesh.org/svnrepo/OpenMesh/trunk/", :using => :svn
 
   depends_on "cmake" => :build
-  depends_on "qt"
+  depends_on "qt" => :optional
```

Comment:
@DomT4 
According to what you have just replied, am I to understand that `depends_on "openssl" => :optional` is equivalent to the following two lines:
```
option "with-openssl"
depends_on "openssl" if build.with "openssl"
```